blog/2024-09-10-pants-2-22/index.mdx

```diff
@@ -8,14 +8,10 @@
 # Pants 2.22.0 is released!
 
 <CaptionedImg src={require("./splash.jpeg").default}>
-  Photo by{" "}
```

Comment:
This was the copy-pasted HTML that unsplash recommends, which our formatter reformatted in a way that broke the rendering (spread it over multiple lines for no reason). Switching to markdown renders correctly after formatting the source.